.devcontainer/devcontainer.json

```diff
@@ -29,6 +29,8 @@
 		"marquesmps.dockerfile-validator",
 		"streetsidesoftware.code-spell-checker",
 		"vscode-icons-team.vscode-icons",
+		"hediet.vscode-drawio",
+		"naumovs.color-highlight",
```

Comment:
I understand your point of view but from developer experience i can see the diagram straight when i open vs code I don't have to find an extension or install drawio. When you say "i didn't want to force people to have it -" i don't really know what is the cost for the end user (extra Mo in the hard drive ?).

For the naumovs.color-highlight extension maybe another is better but it is the same point of view when read the line
` botStore.botStores[bot.botId].isBotOnline ? '&#128994;' : '&#128308;'` see it colorize help me to understand.

The point is : do i may need extra steps to have a development environment that i can straight work with  ?

Reply:
the draw.io diagram is not necessary to work on freqUI (it's also probably outdated as we're no longer using vuex).

the color settings is something that's purely personal - similar to the editor theme.

I could also force my theme on all users - because i think that's the best to work with and get started immediately. It's however my personal view - similar to the color extension.

As such - the extensions necessary to get straight to work are already listed, neither of the 2 additional once are necessary for general development.
they may be helpful for you personally - but that's a personal preference only - not something that will help other users.